cmd/influxd/server_integration_test.go

```diff
@@ -600,6 +600,7 @@ func runTestsData(t *testing.T, testName string, nodes Cluster, database, retent
 		if name == "" {
 			name = tt.query
 		}
+		fmt.Printf("TEST: %d: %s\n", i, name)
```

Comment:
Valid requirement, but you can get this with `go test -v` I believe -- it's what I do.